src/CoreWCF.Kafka.Client/src/CoreWCF/ServiceModel/Channels/KafkaMessageProperty.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+namespace CoreWCF.ServiceModel.Channels;
+
+public sealed class KafkaMessageProperty
+{
+    public static readonly string Name = "CoreWCF.ServiceModel.Channels.KafkaMessageProperty";
+
```

Comment:
With the idea I discussed via chat about making properties injectable, if that's something we're going to pursue, this would be better as a const string rather than a static field.

Reply:
Nevermind for this file, just realized this is the client. But the comment still stands for the service equivalent.